ST_mortality.c

```diff
@@ -204,14 +211,29 @@ void mort_Main( Bool *killed) {
   *killed = _SomeKillage;
 }
 
-/***********************************************************/
+/**
+ * \brief Simulates fires and extirpation.
+ * 
+ * Implements killing of plants through wildfire, prescribed fire, or another event
+ * that would result in mortality in single year (killyr) or
+ * extirpation (extirp) where plants are killed and do not return.
+ * 
+ * If wildfire is simulated it will affect all individuals in all species in all groups.
+ * If prescribed fire is simulated it is possible to specify inputs such that fire will 
+ * only affect one group.
```

Comment:
change all references to "group" here to "rgroup"

Reply:
Addressed by commit 984bd13.